InvenTree/company/migrations/0048_alter_supplierpricebreak_updated.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.14 on 2022-07-17 15:14
```

Comment:
There is no need for two separate migrations here, if it can be helped:

The following steps should merge the two migrations into one.

```
cd InvenTree
python ./manage.py migrate company 0046
rm ./company/migrations/0047_supplierpart_updated.py
rm ./company/migrations/0048_alter_supplierpricebreak_updated.py
python ./manage.py makemigrations company
python ./manage.py migrate company
```